### PR TITLE
EPMRPP-35580 React. Multi action button component

### PR DESCRIPTION
**Pull request by AmsterGet**

### CODE REVIEW

File: app/src/components/buttons/multiActionButton/multiActionButton.jsx [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+          <i className={cx('toggle-icon')}>{Parser(ArrowIcon)}</i>
+        </div>
+        <div className={cx('menu')}>
+          {items.map((item) => (
+            <div
+              key={item.value}
+              className={cx('menu-item', { disabled: item.disabled })}
+              onClick={!item.disabled ? item.onClick : null}
+            >
+              <span>{item.label}</span>
```
Comment (yana95) [inline app/src/components/buttons/multiActionButton/multiActionButton.jsx]: `{item.label}`

File: app/src/components/buttons/multiActionButton/multiActionButton.scss [hunk 2]
```diff
@@ -0,0 +1,148 @@
+.multi-action-button {
+  display: flex;
+  align-items: center;
+  position: relative;
```
Comment (yana95) [inline app/src/components/buttons/multiActionButton/multiActionButton.scss]: https://css-tricks.com/poll-results-how-do-you-order-your-css-properties/

File: app/src/components/buttons/multiActionButton/multiActionButton.scss [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+    }
+  }
+}
+
+.button-part {
+  display: block;
+  height: 100%;
+  padding: 0 10px;
+  border: none;
+  box-sizing: border-box;
```
Comment (yana95) [inline app/src/components/buttons/multiActionButton/multiActionButton.scss]: https://css-tricks.com/poll-results-how-do-you-order-your-css-properties/
Please check all styles

File: app/src/components/buttons/multiActionButton/multiActionButton.jsx (hunk 1 above)
Comment (AmsterGet) [reply app/src/components/buttons/multiActionButton/multiActionButton.jsx]: Fixed

File: app/src/components/buttons/multiActionButton/multiActionButton.scss (hunk 2 above)
Comment (AmsterGet) [reply app/src/components/buttons/multiActionButton/multiActionButton.scss]: Fixed

File: app/src/components/buttons/multiActionButton/multiActionButton.scss (hunk 3 above)
Comment (AmsterGet) [reply app/src/components/buttons/multiActionButton/multiActionButton.scss]: Fixed

File: app/src/components/buttons/multiActionButton/multiActionButton.scss [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+    .menu {
+      display: block;
+    }
+  }
+  &.color-booger {
+    .button-part {
+      background-color: $COLOR--booger;
+
+      &:hover {
+        background-color: darken($COLOR--booger, 15%);
```
Comment (DiscoElevator) [inline app/src/components/buttons/multiActionButton/multiActionButton.scss]: Are there special colors for the disabled buttons in the palette?
https://zpl.io/bokl9Kv - I see here some colors for disabled state, but they are not from our palette ( https://zpl.io/a3qYxMd ). Let's clarify this

File: app/src/components/buttons/multiActionButton/multiActionButton.stories.jsx [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+import { host } from 'storybook-host';
+import { withReadme } from 'storybook-readme';
+import { MultiActionButton } from './multiActionButton';
+import README from './README.md';
+
+const items = [
+  {
+    label: 'Save and post issue',
+    value: 'Post',
+    onClick: action('clicked'),
```
Comment (DiscoElevator) [inline app/src/components/buttons/multiActionButton/multiActionButton.stories.jsx]: Please give different names for actions, to make possible to distinguish them in storybook actions panel

File: app/src/components/buttons/multiActionButton/multiActionButton.scss (hunk 4 above)
Comment (AmsterGet) [reply app/src/components/buttons/multiActionButton/multiActionButton.scss]: I just used colors like in `BigButton` component.
So I need to clarify it with Designer

File: app/src/components/buttons/multiActionButton/multiActionButton.stories.jsx (hunk 5 above)
Comment (AmsterGet) [reply app/src/components/buttons/multiActionButton/multiActionButton.stories.jsx]: Fixed

**Pull request merged**